x/distribution/keeper/allocation.go

```diff
@@ -75,15 +74,11 @@ func (k Keeper) AllocateTokens(ctx sdk.Context, sumPrecommitPower, totalPower in
 	feePool.CommunityPool = feePool.CommunityPool.Add(remaining)
 	k.SetFeePool(ctx, feePool)
 
-	// update outstanding rewards
```

Comment:
this is because now it's tracked per validator, correct?

Reply:
yes, that's right